paper.md

```diff
@@ -2,11 +2,77 @@
 title: "Popper 2.0: A Container-Native Workflow Execution Engine For Testing Complex Applications and Reproducing Scientific Explorations"
 author: Ivo Jimenez, Jayjeet Chakraborty, Arshul Mansoori, Quincy Wofford and Carlos Maltzahn
 abstract: |
- Abstract
+    The problem of reproducibility and replication in scientific research is 
+    quite prevalent to date. Researchers working in fields of computational 
+    science often find it difficult to reproduce experiments from their artifacts 
+    like code, data, diagrams, results, etc. left behind by the previous 
+    researcher. According to a study of 2016 by Nature, among a group of 1576 
+    scientists around 70% of them failed to reproduce each other's experiments. 
+    The code developed on one machine often fail to run in other machines due to 
+    differences in OS, architecture, etc. This is accompanied by the 
+    difficulty of organizing and using the experimental artifacts in the correct 
+    order. Researchers have tried to solve this problem successfully in the past by 
+    building and using workflow engines. With the advent of light-weight virtualization technologies in the form of containers, researchers
+    and developers have built various workflow engines like Argo, Pachyderm, Brigade, etc. for executing workflows inside containers through Kubernetes clusters. This poses an extra overhead of the availability of Kubernetes clusters deployed in the cloud even when such an arrangement is not always necessary. Also, there are workflow engines like Nextflow and Pegasus that do not enforce execution in containers and hence makes the workflows less portable. We introduce Popper, a container-native workflow engine that executes each step of a workflow in its separate dedicated container in a wide range of computing environments. It does not require the presence of a Kubernetes cluster or any cloud Kubernetes service like Google's Kubernetes Engine (GKE), Elastic Kubernetes Services (EKS) from AWS or Microsoft Azure's Kubernetes service.
```

Comment:
> It's difficult to comment on long lines like this. One thing we can do is to define a convention of writing one sentence per line, so that we don't have to struggle with line breaks before periods (or multiple sentences in the same line) when commenting on git diffs.

Sure, one sentence per line is best.